skfem/assembly/__init__.py

```diff
@@ -7,7 +7,10 @@
 >>> m = fem.MeshTri()
 >>> e = fem.ElementTriP1()
 >>> m
-Triangular mesh with 4 vertices and 2 elements.
+<skfem MeshTri1 object>
+  Number of elements: 2
+  Number of vertices: 4
+  Number of nodes: 4
```

Comment:
This now better highlights the difference (if any) in the number of vertices (of the underlying topology) and the number of nodes in the finite element field describing the geometry.

Reply:
It also shows `boundaries` and `subdomains`. You can try it out in REPL.

Reply:
Nice, very nice; very useful.  I will use this a lot, a simple `print(mesh)` will be a standard in my logging, I think.  Ta.